WalletWasabi.Tests/UnitTests/WabiSabi/Backend/UtxoPrisonTests.cs

```diff
@@ -30,22 +31,22 @@ public void PrisonChangeTracking()
 			// Make sure we set them to the past so the release method that looks at the time evaluates to true.
 			var past = DateTimeOffset.UtcNow - TimeSpan.FromMilliseconds(2);
 
-			var guid1 = Guid.NewGuid();
-			p.Punish(new Inmate(BitcoinFactory.CreateOutPoint(), Punishment.Banned, past, guid1));
+			var id1 = new uint256(1234);
```

Comment:
Same question here, Zero is not good?

Reply:
Yes, in this case there are two ids that need to be different so `uint256.Zero` and `uint256.One` are okay. I will change them.

Reply:
I changed these hardcoded values by `BitcoinFactory.CreateUInt256()`.